DocSum/README.md

```diff
@@ -156,21 +151,9 @@ Two ways of consuming Document Summarization Service:
 1. Use cURL command on terminal
 
    ```bash
-   #Use English mode (default).
-   curl http://${host_ip}:8888/v1/docsum \
-       -H "Content-Type: multipart/form-data" \
-       -F "messages=Text Embeddings Inference (TEI) is a toolkit for deploying and serving open source text embeddings and sequence classification models. TEI enables high-performance extraction for the most popular models, including FlagEmbedding, Ember, GTE and E5." \
-       -F "max_tokens=32" \
-       -F "language=en" \
-       -F "stream=true"
-
-   #Use Chinese mode.
-   curl http://${host_ip}:8888/v1/docsum \
-       -H "Content-Type: multipart/form-data" \
-       -F "messages=2024年9月26日，北京——今日，英特尔正式发布英特尔® 至强® 6性能核处理器（代号Granite Rapids），为AI、数据分析、科学计算等计算密集型业务提供卓越性能。" \
-       -F "max_tokens=32" \
-       -F "language=zh" \
-       -F "stream=true"
+   curl -X POST http://${host_ip}:8888/v1/docsum \
+        -H "Content-Type: application/json" \
+        -d '{"type": "text", "messages": "Text Embeddings Inference (TEI) is a toolkit for deploying and serving open source text embeddings and sequence classification models. TEI enables high-performance extraction for the most popular models, including FlagEmbedding, Ember, GTE and E5."}'
```

Comment:
Shouldn't the other parameters be also given, at least language?

And because this is "document summary" application, I would really like example of an actual document (file) being summarized, instead of some ridiculously short text.

(This comment applies to all docsum usage examples in this PR, not just the one in this README.)

Reply:
Thanks for your feedback. 
You can upload pdf, doc ,docx, audio and video documents. 

![image](https://github.com/user-attachments/assets/ef477b73-617f-40c1-a67c-a1829e0d403b)

Parameter (language, llm_model, whisper model etc.) setting will be included in v.1.2

Reply:
That's the UI, I'm not really interested about that, but the megaservice API for uploading such files...